testing/python/fixture.py

```diff
@@ -1614,6 +1614,22 @@ def test_issue(foo, foobar):
         reprec = testdir.inline_run()
         reprec.assertoutcome(passed=9)
 
+    @pytest.mark.parametrize('param_args', ["'fixt, val'", "'fixt,val'", "['fixt', 'val']", "('fixt', 'val')"])
+    def test_override_parametrized_fixture_issue_979(self, testdir, param_args):
+        testdir.makepyfile("""
+            import pytest
+
+            @pytest.fixture(params=[1, 2])
+            def fixt(request):
+                return request.param
+
+            @pytest.mark.parametrize(%s, [(3, 'x'), (4, 'x')])
```

Comment:
`.format` isn't used anywhere in this file, and `%`-formatting is used 23 times (which is why I used that) - do you really want to introduce that inconsistency?